src/test/java/org/fogbowcloud/manager/OrderStateTransitionerTest.java

```diff
@@ -44,10 +42,10 @@ public void setUp() {
     public void tearDown() {
         SharedOrderHolders instance = SharedOrderHolders.getInstance();
 
```

Comment:
avoid /** comments within methods. /** is for javadoc

Reply:
Ok, done.